Sources/JWT/Signers/RSASigner+X509.swift

```diff
@@ -0,0 +1,30 @@
+import CTLS
+
+typealias CX509Key = UnsafeMutablePointer<X509>
+
+extension RSAKey {
+    public init(x509Cert cert: Bytes) throws {
+        guard
+            let cert = cert.withUnsafeBufferPointer({ rawKeyPointer -> CX509Key? in
```

Comment:
Break up the guard into multiple guards, yes. (I would consider memory safety close to infinitely more important than compactness.)